Install a copy of the builtin gstomx.conf defaults to /etc which has the
plugin library paths adjusted from /usr/lib/arm-linux-gnueabihf/tegra to
/usr/lib/tegra.

diff -u -uNr nvidia-gst-21.3.0.orig/etc/xdg/gstomx.conf nvidia-gst-21.3.0/etc/xdg/gstomx.conf
--- nvidia-gst-21.3.0.orig/etc/xdg/gstomx.conf	1970-01-01 01:00:00.000000000 +0100
+++ nvidia-gst-21.3.0/etc/xdg/gstomx.conf	2015-07-23 14:10:57.594610847 +0200
@@ -0,0 +1,71 @@
+[omxmpeg4videodec]
+type-name=GstOMXMPEG4VideoDec
+core-name=/usr/lib/tegra/libnvomx.so
+component-name=OMX.Nvidia.mp4.decode
+rank=266
+
+[omxh264dec]
+type-name=GstOMXH264Dec
+core-name=/usr/lib/tegra/libnvomx.so
+component-name=OMX.Nvidia.h264.decode
+rank=266
+
+[omxvp8dec]
+type-name=GstOMXVP8Dec
+core-name=/usr/lib/tegra/libnvomx.so
+component-name=OMX.Nvidia.vp8.decode
+rank=266
+
+[omxmpeg2videodec]
+type-name=GstOMXMPEG2VideoDec
+core-name=/usr/lib/tegra/libnvomx.so
+component-name=OMX.Nvidia.mpeg2v.decode
+rank=266
+
+[omxh263dec]
+type-name=GstOMXH263Dec
+core-name=/usr/lib/tegra/libnvomx.so
+component-name=OMX.Nvidia.h263.decode
+rank=266
+
+[omxwmvdec]
+type-name=GstOMXWMVDec
+core-name=/usr/lib/tegra/libnvomx.so
+component-name=OMX.Nvidia.vc1.decode
+rank=266
+
+[omxaacdec]
+type-name=GstOMXAACDec
+core-name=/usr/lib/tegra/libnvomx.so
+component-name=OMX.Nvidia.eaacp.decoder
+rank=0
+
+[omxmpegaudiodec]
+type-name=GstOMXMPEGAUDIODec
+core-name=/usr/lib/tegra/libnvomx.so
+component-name=OMX.Nvidia.mp3.decoder
+rank=0
+
+[omxh264enc]
+type-name=GstOMXH264Enc
+core-name=/usr/lib/tegra/libnvomx.so
+component-name=OMX.Nvidia.h264.encoder
+rank=266
+
+[omxvp8enc]
+type-name=GstOMXVP8Enc
+core-name=/usr/lib/tegra/libnvomx.so
+component-name=OMX.Nvidia.vp8.encoder
+rank=266
+
+[nvhdmioverlaysink]
+type-name=GstNvHDMIOverlaySink
+core-name=/usr/lib/tegra/libnvomx.so
+component-name=OMX.Nvidia.render.hdmi.overlay.yuv420
+rank=256
+
+[nvoverlaysink]
+type-name=GstNvOverlaySink
+core-name=/usr/lib/tegra/libnvomx.so
+component-name=OMX.Nvidia.std.iv_renderer.overlay.yuv420
+rank=266
